viam_cartographer.go

```diff
@@ -355,9 +367,15 @@ func parseCartoAlgoConfig(configParams map[string]string, logger logging.Logger)
 			}
 			cartoAlgoCfg.MinRange = float32(fVal)
 		case "min_range_meters":
-			fVal, err := strconv.ParseFloat(val, 32)
-			if err != nil {
-				return cartoAlgoCfg, err
+			var fVal float64
+			var err error
+			if val == "" {
+				fVal = float64(defaultCartoAlgoCfg.MinRange)
+			} else {
+				fVal, err = strconv.ParseFloat(val, 32)
```

Comment:
...line 373 has a float64, but line 375 seems to expect a 32-bit float. Are you sure about this? ("Yes" is a very believable answer!)

Reply:
yeah, the issue is ParseFloat always returns a float64, evene when you tell it to use 32 bits. Setting up the help functions to handle this better tho